.pre-commit-hooks.yaml

```diff
@@ -16,7 +16,7 @@
   name: Google Java Formatter
   description: Runs Google Java Formatter over Java source files
   entry: pretty-format-java
-  language: python
+  language: docker
```

Comment:
My intention here isn't to have this piece committed, but to illustrate how this formatter can be run inside Docker, and to open discussion about how to provide a containerized version of this in case anyone else runs into a similar issue